conf/arm64.src

```diff
@@ -0,0 +1,8 @@
+# This is on YunoHost server just to avoid the file from disappearing
+# Original source is https://releases.wekan.team/
+SOURCE_URL=https://releases.wekan.team/raspi3/wekan-2.94-arm64-bundle.tar.gz
```

Comment:
This file is not hosted in https://build.yunohost.org/apps/ to avoid it disappearing ...

Reply:
Yes, I know I haven't access to https://build.yunohost.org/apps/ , so if you can upload the two files ^^ I will make the changes on the src files

Reply:
Just seen them, I will update :)

Reply:
No worries, took care of it, that's my contribution to the maintenance of this app :sweat_smile: